concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1514,6 +1514,31 @@ jobs:
 
                   cf push
 
+        - task: update-kibana-timezone
+          config:
+            platform: linux
+            image: docker:///ruby#2.2-slim
```

Comment:
note that after https://github.com/alphagov/paas-cf/pull/381 is merged, this should be changed to image_resource